### DISCUSSION TITLE
1.0.0-beta.6

### DISCUSSION THREAD

**Discussion by kennethnym** [MrCreeper1008/SpotlightLauncher#33] [Announcements]
- Quick notes widget:
  - The add note text field is now cleared automatically when a new note is added
  - All notes overlay:
    - Top padding no longer clips content
    - Add note button no longer blocks the note cards
    - Content will now animate with IME/keyboard to avoid the IME blocking the content
- Clock:
  - There is now a refresh button next to the weather
  - Clock size can now be adjusted in settings: Settings > Clock > Clock size.
    There are 5 sizes to choose from: giga small, small, normal (default), big, giga big.
- Search results:
  - Search results can now be dismissed by pressing the back button

<hr /><em>This discussion was created from the release <a href='https://github.com/kennethnym/StarlightLauncher/releases/tag/1.0.0-beta.6'>1.0.0-beta.6</a>.</em>

**Comment by MilesRobinson764** [comment]
Nice!
